functions/src/import/index.ts

```diff
@@ -95,3 +100,19 @@ rl.question(`\nWARNING: The back fill process will index your entire collection
   rl.close();
 });
 
+const doesPathMatchConfigCollectionPath = (path: string): boolean => {
+  const pathSegments = path.split('/');
+  const collectionPathSegments = config.collectionPath.split('/')
+
+  // Iterate over the config collection path by each subcollection only
+  for (let i = 0; i < collectionPathSegments.length; i+=2) {
+    const configSegment = collectionPathSegments[i];
+    const pathSegment = pathSegments[i];
+
+    if (pathSegment != configSegment) {
+      return false;
+    }
+  }
+  
+  return true;
```

Comment:
```suggestion
  return collectionPathSegments.every(
    (configSegment, i) => configSegment === pathSegments[I]
  );
```

this can be simplified